lib/FrontendTool/FrontendTool.cpp

```diff
@@ -504,10 +504,17 @@ createOptRecordFile(StringRef Filename, DiagnosticEngine &DE) {
   return File;
 }
 
+struct PostSILGenInputs {
```

Comment:
"Conflicts" in this commit message too ("Thread the ModuleOrSourceFile of each…")

Reply:
Removed the conflicts parts of the commit messages.